### ISSUE TITLE
TypeError: Cannot read property 'months' of undefined

### ISSUE THREAD

**Issue by thedist** [jctrvlr/twitchPS#2]
TypeError when there is a subscription event
main.js - 249     `  "months" : message.data.message.data.months,`
I think should be `  "months" : message.data.message.months,`

**Comment by jctrvlr** [comment]
Okay looking into it now. Sorry about that, I assumed the message would be similar to bits. This is what i get for assuming haha.

**Comment by jctrvlr** [comment]
Should work now! Closing this issue for now! Thank you for reporting my stupid typo.

**Comment by jctrvlr** [comment]
Reopened because error is still occurring. Looking into it now

**Comment by jctrvlr** [comment]
Nevermind it is fixed. I misunderstood the new system. The user has the option to either annouce or withhold from announcing their sub in chat. This creates confusion because the old system that announces subscriptions in chat is currently being deprecated, but before it is completely phased out.

TLDR; see https://discuss.dev.twitch.tv/t/subscriptions-beta-changes/10023/37
